src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseTypeDefinition.java

```diff
@@ -37,6 +37,10 @@
     private String printableDocumentsUrl;
     @JsonProperty("acls")
     private List<AccessControlList> accessControlLists;
+    @JsonProperty("callbackGetCaseUrl")
+    private String callbackGetCaseUrl;
+    @JsonProperty("retriesGetCaseUrl")
```

Comment:
@JsonProperty("callback_get_case_url")
    private String callbackGetCaseUrl;
    @JsonProperty("retries_get_case_url")
    private List<Integer> retriesGetCaseUrl;
    
    These are json Properties in Defintion store

Reply:
done